dev/unix/volta-install.sh

```diff
@@ -23,7 +23,7 @@ download_release_from_repo() {
   local download_file="$tmpdir/$filename"
   local archive_url="$(release_url)/download/v$version/$filename"
 
-  curl --progress-bar --show-error --location --fail "$archive_url" --output "$download_file" && echo "$download_file"
+  curl --progress-bar --show-error --location --fail "$archive_url" --output "$download_file" --write-out '%{filename_effective}'
```

Comment:
Alternatively, we can call `curl --disable ...` to disable reading the default config file. The downside is that users may have good reasons to change settings like proxy, etc.

Reply:
Agreed, I don't think we want to disable the user's config, so using the `--write-out` parameter makes sense.